llm/openrlhf/openrlhf_sft.yaml

```diff
@@ -0,0 +1,61 @@
+# Adapted from https://openrlhf.readthedocs.io/en/latest/rl.html#supervised-fine-tuning
+# To run simply run the following command in a SkyPilot environment (with GCP configured):
+# WANDB_TOKEN=xxx HF_TOKEN=xxx sky launch -c open-rlhf-sft openrlhf_sft.yaml --secret WANDB_TOKEN --secret HF_TOKEN
+
+resources:
+  infra: gcp
+  accelerators: A100:8
+  use_spot: True
+  autostop:
+    idle_minutes: 2 # Auto-stop after 2 minutes of idleness.
+
+envs:
+  WANDB_TOKEN: null # Pass with `--secret WANDB_TOKEN` in CLI
+  HF_TOKEN: null # Pass with `--secret HF_TOKEN` in CLI
+
+file_mounts:
+  /checkpoints:
+    name: openrlhf-sft-checkpoints
+    store: gcs
+
+setup: |
+  sudo apt update
+  docker pull nvcr.io/nvidia/pytorch:25.02-py3
+  docker run --name openrlhf_tmp --runtime=nvidia -v $PWD:/openrlhf nvcr.io/nvidia/pytorch:25.02-py3 bash -c "
+          pip uninstall xgboost transformer_engine flash_attn pynvml opencv-python-headless -y
+
+          pip install git+https://github.com/OpenRLHF/OpenRLHF.git@1bfcc334692f4d1e0ec0817465d3d9d495fb162f
+  "
+  docker commit openrlhf_tmp openrlhf:custom
+  docker rm openrlhf_tmp

```

Comment:
Have we considered to set the `image_id` with the docker image directly? https://docs.skypilot.co/en/latest/examples/docker-containers.html#using-containers-as-runtime-environments